src/js/fluxcontroller.js

```diff
@@ -202,6 +219,32 @@ define(function (require, exports, module) {
     });
 
     /**
+     * Initialize all action functions 
+     *
+     * @private
+     */
+    FluxController.prototype._initAllActions = function () {
+        this._allActions = {};
+        this._actionModules = new Map();
```

Comment:
I think this should be called `_actionModuleNames`? It maps actions to action module names, right? The JSDoc above should be clarified too.